screening/openg2p.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "OpenG2P",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://docs.openg2p.org/"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.openg2p.org/"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "nonPIIAccessMechanism": "Yes"
```

Comment:
This should describe those mechanisms, a simple `Yes` answer is not sufficient here. We need to go back to them and ask for additional information.

Reply:
A more descriptive  mechanism for accessing non PII data was availed in this [commit](https://github.com/unicef/publicgoods-candidates/pull/112/commits/a7db56c683f65fdc3ed8db3a42cee52bd3e561a1)